test/unit/rules/lint-no-action-helper-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+const ERROR_MESSAGE = require('../../../lib/rules/lint-no-action-helper').ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'no-action-helper',
+
+  config: true,
+
+  good: [
+    '{{#let (fn this.foo bar) as |action|}}<Component @baz={{action}} />{{/let}}',
+    '<MyScope as |action|><Component @baz={{action}} /></MyScope>',
+    '<button {{on "submit" @action}}>Click Me</button>',
+    '<button {{on "submit" this.action}}>Click Me</button>',
+  ],
+
+  bad: [
```

Comment:
Can you add `bad` examples for:

* `<button onclick={{action 'foo'}}></button>`